README.md

```diff
@@ -28,71 +29,84 @@ There are many benefits in running your application as a unikernel: for more inf
 - 🧰 Go SDK for building unikernels programmatically; and
 - 🚀 _much more!_
 
+
 ## Installation
 
-You can quickly and easily install KraftKit using the interactive installer.  Simply run the following command to get started: 
+You can quickly and easily install KraftKit using the interactive installer.
+Simply run the following command to get started:
 
 ```shell
 curl --proto '=https' --tlsv1.2 -sSf https://get.kraftkit.sh | sh
 ```
 
 Alternatively, you can download the binaries from the [releases pages](https://github.com/unikraft/kraftkit/releases).
 
+See [additional installation instructions](https://unikraft.org/docs/cli/install).
+
+See also the [hacking documentation on how to build KraftKit from source](https://unikraft.org/docs/cli/hacking).
+
+
 ### Container build environment
 
-KraftKit ships a container build environment which you can use instead of installing any dependencies directly on your host.
-It includes the `kraft` binary as well as all the additional tools and libraries for building Unikraft unikernels.
-Simply attach a working directory on your host as a mount path volume mapped to `/workspace`, e.g.:
+KraftKit ships a container build environment which you can use instead of installing any dependencies directly on your host. It includes the `kraft` binary as well as all the additional tools and libraries for building Unikraft unikernels. Simply attach a working directory on your host as a mount path volume mapped to `/workspace`, e.g.:
```

Comment:
Every sentence on a new line ;)

Applies for all other cases too

Reply:
Ah should have been, whoops!